src/EnergyPlus/OutsideEnergySources.cc

```diff
@@ -386,12 +424,12 @@ void OutsideEnergySourceSpecs::size(EnergyPlusData &state)
         }
     } else {
         if (this->NomCapWasAutoSized && state.dataPlnt->PlantFirstSizesOkayToFinalize) {
-            ShowSevereError(state, format("Autosizing of District {} nominal capacity requires a loop Sizing:Plant object", typeName));
-            ShowContinueError(state, format("Occurs in District{} object={}", typeName, this->Name));
+            ShowSevereError(state, format("Autosizing of {} nominal capacity requires a loop Sizing:Plant object", typeName));
+            ShowContinueError(state, format("Occurs in {} object={}", typeName, this->Name));
             ErrorsFound = true;
         }
         if (!this->NomCapWasAutoSized && this->NomCap > 0.0 && state.dataPlnt->PlantFinalSizesOkayToReport) {
-            BaseSizer::reportSizerOutput(state, "District" + typeName, this->Name, "User-Specified Nominal Capacity [W]", this->NomCap);
+            BaseSizer::reportSizerOutput(state, format("{}", typeName), this->Name, "User-Specified Nominal Capacity [W]", this->NomCap);
```

Comment:
@Myoldmopar I missed this one, can be simplified to `state, typeName, this->Name, ...` but it works as-is.